ktor-server/ktor-server-test-host/build.gradle

```diff
@@ -26,8 +26,6 @@ kotlin.sourceSets.jvmMain.dependencies {
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-debug:$coroutines_version"
 }
 
-if (!project.ext.ideaActive) {
```

Comment:
Why do we need ideaActive at all?

Reply:
or why did we need*

Reply:
It was a bug in old kotlin plugin, the didn't support custom Gradle configurations